docs/en/observability/monitor-azure-agent.asciidoc

```diff
@@ -0,0 +1,381 @@
+. Save the diagnostic settings.
+
+[discrete]
+[[azure-elastic-agent-configure-logs-integration]]
+==== Configure the Azure Logs integration to collect activity logs
+
+Now that activity logs are streaming into the event hub, you can configure
+the Azure activity log integration to ingest the logs.
+
+To add the integration: 
+
+. Go to the {kib} home page and click **Add integrations**.
+
+. In the query bar, search for **Azure activity logs** and select the Azure
+activity logs integration to see more details about it.
+
+. Click **Add Azure activity logs**.
+
+. Under Integration settings, configure the integration name and optionally add
+a description.
+
+. Specify values for all the required fields. For more information about these
+settings, refer to the {integrations-docs}/activitylogs[Azure activity logs]
+documentation.
++
+--
+[horizontal]
+**Eventhub**:: The name of the event hub you created earlier.
+
+**Connection String**:: The connection string primary key of the event hub
+namespace. To learn how to get the connection string, refer to
+https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string[Get an Event Hubs connection string]
+in the Azure documentation.
++
+TIP: Instead of copying the connection string from the RootManageSharedAccessKey
+policy, you should create a new shared access policy (with permission to listen)
+and copy the connection string from the new policy.
+
+**Storage account**:: The name of a blob storage account that {agent} can use
+to store information about logs consumed by the agent. You can use the same
+storage account container for all integrations.
+
+**Storage account key**:: A valid access key defined for the storage account.
+--
++
+[role="screenshot"]
+image::images/agent-tut-azure-activity-log-settings.png[Screenshot of integration settings for Azure activity logs]
+
+. Make sure the **Collect Azure activity logs from Event Hub** selector is
+turned on.
+
+. Under **Existing hosts**, select the agent policy that created earlier.
+
+. Save and deploy the integration.
++
+This step takes a minute or two to complete. When it's done, you'll have an
+agent policy that contains the Azure activity logs configuration plus the
+billing metrics configuration. The deployed {agent} will pick up the policy
+change and start sending Azure activity logs to {es}.
+
+[discrete]
+[[azure-elastic-agent-visualize-azure-logs]]
+=== Step 5: Visualize Azure activity logs
+
+Now that logs are streaming into {es}, you can visualize them in {kib}. To see
+the raw logs, open the main menu in {kib}, then click **Logs**. Notice that you
+can filter on a specific data stream. This example uses
+`data_stream.dataset : "azure.activitylogs"` to show Azure activity logs:
+
+[role="screenshot"]
+image::images/agent-tut-azure-activity-logs.png[Screenshot of Logs app showing Azure activity logs]
+
```

Comment:
Nit: we can avoid the "failed to find message" by suggesting users change the Observability > Logs > Stream settings:

![CleanShot 2022-10-10 at 17 43 02@2x](https://user-images.githubusercontent.com/25941/194905241-afcd4407-e208-4a5b-872e-5fdf60920e01.png)


![CleanShot 2022-10-10 at 17 43 17@2x](https://user-images.githubusercontent.com/25941/194905272-09a51f77-057a-45e6-a117-702b6dda6fc7.png)

Reply:
I'll add that as a TIP.  I'll leave the screen capture because I think it's actually useful to explain why this happens.

Reply:
Yeah, it's the right call since it's what most users will get OOTB.